How do I add parse and bolts framework

anirudhamahale opened the pull request:
Hi thanks alot for the SDK but how do I add parse and bolts framework?

I didn't find any framework with such name in your project. 

so please help me.

Juddster committed:
Modified to explain migration to Backand

Juddster committed:
fix a couple typos

danielnaim committed:
removed licence

Juddster commented:
Hi there,

Thanks for your interest in this SDK. First of all, this repo was a temporary one. We moved it over to here: https://github.com/backand/Backand-SDK-iOS-OSX https://github.com/backand/Backand-SDK-iOS-OSX

Secondly, we are not quite ready yet. We didn’t push the built binaries to the git repo yet. You are welcome to download our build of Parse-iOS.zip from this temporary link: https://www.dropbox.com/s/gbczaobi6k7dwxd/Parse-iOS.zip?dl=0 https://www.dropbox.com/s/gbczaobi6k7dwxd/Parse-iOS.zip?dl=0

I’m CC Daniel, so he can followup with you when we have everything in place.

For now, see the updated Readme at the new repro I mentioned above. If you have an existing Project that uses the Parse SDK as a built framework, you can try out ours and see how it goes. If you are using the pod version, give us a few more days. We’ll have that working too.

Your feedback will be very much appreciated!

Thanks,
Judd

P.S. Feel free to email me directly (judd@contity.com mailto:judd@contity.com) as I am going to delete the temporary repo you used to contact me.

> On Mar 4, 2016, at 7:53 AM, anirudhamahale notifications@github.com wrote:
> 
> Hi thanks alot for the SDK but how do I add parse and bolts framework?
> 
> I didn't find any framework with such name in your project.
> 
> so please help me.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/Juddster/Parse-SDK-iOS-OSX/pull/1 https://github.com/Juddster/Parse-SDK-iOS-OSX/pull/1
> Commit Summary
> 
> Modified to explain migration to Backand
> fix a couple typos
> removed licence
> File Changes
> 
> M README.md https://github.com/Juddster/Parse-SDK-iOS-OSX/pull/1/files#diff-0 (113)
> Patch Links:
> 
> https://github.com/Juddster/Parse-SDK-iOS-OSX/pull/1.patch https://github.com/Juddster/Parse-SDK-iOS-OSX/pull/1.patch
> https://github.com/Juddster/Parse-SDK-iOS-OSX/pull/1.diff https://github.com/Juddster/Parse-SDK-iOS-OSX/pull/1.diff
> —
> Reply to this email directly or view it on GitHub https://github.com/Juddster/Parse-SDK-iOS-OSX/pull/1.

Juddster committed:
updated pod spec

Juddster committed:
rename Parse.podspec to Backand.podspec
replace #import <Parse/xxx.h> with #import "xxx.h" in the entire code tree because it prevent the podspec from passing validation

Juddster committed:
Add CocoaPods installation instructions to README.md
